src/Assert.php

```diff
@@ -590,6 +596,70 @@ public static function isInstanceOfAny($value, array $classes, $message = '')
         ));
     }
 
+    /**
+     * @param object|string $value
+     * @param string        $class
+     * @param string        $message
+     *
+     * @throws InvalidArgumentException
+     */
+    public static function isAOf($value, $class, $message = '')
+    {
+        static::string($class, 'Expected class as a string. Got: %s');
+
+        if (!is_a($value, $class, is_string($value))) {
```

Comment:
One last thing before i merge this. Can you change all function calls, like `is_a` with `\is_a`?

Reply:
done